### PR TITLE
Fix two small crashes

### PR DESCRIPTION
**Pull request by vext01**
Two small fixes I had to apply to get the right graphs out of the plotting script for my slides.

See commit messages for more info.

### CODE REVIEW

File: bin/plot_krun_results [hunk 1]
```diff
@@ -851,7 +851,8 @@ def get_data_dictionaries(json_files, benchmarks=[], wallclock_only=False,
         instr_data = False
         filename_root = filename[:-len('_results.json.bz2')]
         instr_dir = os.path.dirname(filename) + filename_root + '_instr_data'
-        data_dictionary['instr_data'] = dict()
+        if 'instr_data' not in data_dictionary:
+            data_dictionary['instr_data'] = dict()
```
Comment (snim2) [inline bin/plot_krun_results:855]: OK, does the same think happen with cycles data or any other data?

File: bin/plot_krun_results [hunk 2]
```diff
@@ -264,7 +264,11 @@ def main(is_interactive, data_dcts, plot_titles, window_size, outfile,
                 ret = list()
                 for i in xrange(len(page)):
                     if page[i]:
-                        ret.append(data[i])
+                        try:
+                            ret.append(data[i])
+                        except IndexError:
+                            # absent data
```
Comment (snim2) [inline bin/plot_krun_results]: s/absent data/Absent data./g

File: bin/plot_krun_results (hunk 1 above)
Comment (vext01) [reply bin/plot_krun_results:855]: Not that I observed.

File: bin/plot_krun_results (hunk 2 above)
Comment (vext01) [reply bin/plot_krun_results]: fixed.

**Comment by vext01** [comment]
ok to squash?

**Comment by snim2** [comment]
Yes, you've tested this?

**Comment by snim2** [comment]
Fixes #303

**Comment by vext01** [comment]
It was (informally) tested by making plots for my slides. I used a fair few different options.

**Comment by snim2** [comment]
OK

**Comment by vext01** [comment]
splat.

**Review by snim2 — APPROVED** [review]

**Pull request merged**